src/test/regress/sql/multi_unsupported_worker_operations.sql

```diff
@@ -0,0 +1,203 @@
+--
+-- MULTI_METADATA_SYNC
+--
+
+-- Tests for ensuring unsupported functions on workers error out.
+
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1270000;
+ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1270000;
+
+-- Prepare the environment
```

Comment:
Actually, with the second check, placement id has no effect on this test, therefore removed it for now.